woocommerce/woocommerce-ios
# [HACK Week] Order Navigation

## Description

With this PR we implement order navigation within the order details. With this, users can navigate to previous or next orders without the need to go back to the order list and tap on an order again. This was requested a couple of times through AppStore/Google Play reviews.
If the Order Detail screen is presented through a notification or shortcut these buttons are hidden.

### Known caveats
With this implementation, the user can only visit those orders that were retrieved when loading the order list. That means that if there's only one page loaded the user will be bound to those orders. Even with that and given the size of our pages (25) the users can still benefit from it.

## Testing instructions


1. Go to orders
2. Tap on an order
3. See that the open order is the one that was selected previously
4. See that you can navigate to previous and next orders, but when you reach a bound -upper or lower- the matching button is disabled.

## Screenshots


https://github.com/woocommerce/woocommerce-ios/assets/1864060/652950a4-a906-4a1b-9c8e-39de934a66c1



---
